Makefile

```
@@ -96,6 +98,8 @@ kind-load-local-images:
 	$(KIND) load docker-image $(MANAGER_IMG) --name $(KIND_CLUSTER)
 	$(KIND) load docker-image $(RENDERER_IMG) --name $(KIND_CLUSTER)
 	$(KIND) load docker-image $(DISCOVERY_IMG) --name $(KIND_CLUSTER)
+  $(KIND) load docker-image localhost/local/solar-ui:$(TAG) --name $(KIND_CLUSTER)
```

checkmake (0.3.2)

[warning] Target "$(KIND) load docker-image localhost/local/solar-ui" should be declared PHONY.

(phonydeclared)